packages/react-aria-components/docs/ListBox.mdx

```diff
@@ -842,8 +842,13 @@ function DraggableListBox() {
 Dropping on the ListBox as a whole can be enabled using the `onRootDrop` event. When a valid drag hovers over the ListBox, it receives the `isDropTarget` state and can be styled using the `[data-drop-target]` CSS selector.
 
 ```tsx example
+interface Item {
```

Comment:
I feel like this should be a type we export since it's used a couple times, but it's just for examples to show what to do.